Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -33,20 +35,63 @@ public GameObject ThumbRoot
             }
         }
 
-        [Range(0, 1)]
+        #region MyValue 
+        public int MinValue = -4;
```

Comment:
We typically recommend adding a serialized backing field for Inspector visibility, with a public property if code access is needed outside this script : https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#do-9